### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone or `ldd`. Also, I think you can omit `AFL_SKIP_CPUFREQ`. `@@` is not understood outside of the Mayhemfile context.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+image: ghcr.io/capuanob/bison-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image field. Tests -> testsuite.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**